clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10728,7 +10728,7 @@ static void getSPIRVBackendOpts(const llvm::opt::ArgList &TCArgs,
 static void getNonTripleBasedSPIRVTransOpts(Compilation &C,
                                             const llvm::opt::ArgList &TCArgs,
                                             ArgStringList &TranslatorArgs) {
-  TranslatorArgs.push_back("-spirv-max-version=1.4");
+  TranslatorArgs.push_back("-spirv-max-version=1.5");
```

Comment:
What is the behavior for values greater than `1.5`? Will there be a diagnostic emitted?

Reply:
Yes, `llvm-spirv` itself checks for the supported version. Now the latest available is `1.6`, but it would be introduced later in a separate PR

Reply:
Can a test case be added to verify the above behavior?

Reply:
>Can a test case be added to verify the above behavior?

This behavior is controlled by the translator and the appropriate tests are available there.